js/__internal/grids/grid_core/search/m_search.ts

```diff
@@ -232,8 +233,10 @@ export const searchModule = {
 
         _getStringNormalizer() {
           const isCaseSensitive = this.option('searchPanel.highlightCaseSensitive');
-          return function (str) {
-            return isCaseSensitive ? str : str.toLowerCase();
+          const langParams = this.getController('data')?.getDataSource?.()?.loadOptions?.()?.langParams;
```

Comment:
This line of code looks overcomplicated.
Let's make it simple:

1) Let's move DataController to the field of rowsView (I guess):
```typescript
  init() {
    this._dataController = this.getController('data');
    // other code
  }
```
2) Let's add lines:
```typescript
// Do we need the "?" after the getDataSource method?
const dataSource = this._dataController?.getDataSource?.();
// Do we need the "?" after the loadOptions method?
const loadOptions = dataSource?.loadOptions?.();
const langParams = loadOptions?.langParams;
```